hazelcast/src/main/java/com/hazelcast/internal/nio/tcp/TcpIpNetworkingService.java

```diff
@@ -256,20 +255,19 @@ public void scheduleDeferred(Runnable task, long delay, TimeUnit unit) {
     }
 
     private void startAcceptor() {
-        if (acceptor != null) {
-            logger.warning("TcpIpAcceptor is already running! Shutting down old acceptor...");
+        if (acceptorRef.get() != null) {
```

Comment:
The `startAcceptor() `is racy if called concurrently. Not a problem of current PR though.

Reply:
It's called from a synchronized method only, so currently it's not possible to call it concurrently.